lib/V8/v8-utils.cpp

```diff
@@ -21,6 +21,9 @@
 /// @author Dr. Frank Celler
 ////////////////////////////////////////////////////////////////////////////////
 
+#include <chrono>
+#define arangodb_have_cpp_lib_chrono 1
```

Comment:
we should remove this define, and just use `#include <chrono>` in the header where it is now used.